Xamarin.Forms.Platform.Android/Renderers/FloatingActionButtonRenderer.cs

```diff
@@ -0,0 +1,337 @@
+﻿using Android.Content;
+using Android.Content.Res;
+using Android.Support.V4.View;
+using Android.Views;
+using System;
+using System.ComponentModel;
+using Xamarin.Forms.Internals;
+using AView = Android.Views.View;
+using AFloatingButton = global::Android.Support.Design.Widget.FloatingActionButton;
+using Xamarin.Forms.Platform.Android.FastRenderers;
+using Android.Graphics.Drawables;
+using Android.Widget;
+
+namespace Xamarin.Forms.Platform.Android
+{
+	public class FloatingActionButtonRenderer : AFloatingButton,
+		IVisualElementRenderer, IViewRenderer, ITabStop,
+		AView.IOnAttachStateChangeListener, AView.IOnFocusChangeListener, AView.IOnClickListener, AView.IOnTouchListener
+	{
+		const float SmallSize = 44;
```

Comment:
Is 44 the right size?  Should it be 40?
https://developer.android.com/reference/com/google/android/material/floatingactionbutton/FloatingActionButton.html#SIZE_MINI

Reply:
Yes, you're right, Shane: https://material.io/design/components/buttons-floating-action-button.html#anatomy
FAB containers come in two sizes:
- Default (56 x 56dp)
- Mini (40 x 40dp)

I have applied changes in sizes.